com.unity.multiplayer.mlapi/Runtime/Spawning/NetworkSpawnManager.cs

```diff
@@ -142,13 +153,21 @@ internal void ChangeOwnership(NetworkObject networkObject, ulong clientId)
 
             networkObject.OwnerClientId = clientId;
 
-            using (var buffer = PooledNetworkBuffer.Get())
-            using (var writer = PooledNetworkWriter.Get(buffer))
+            ulong[] clientIds = NetworkManager.ConnectedClientsIds;
+            var messageQueueContainer = NetworkManager.MessageQueueContainer;
+            var context = messageQueueContainer.EnterInternalCommandContext(
```

Comment:
**Nit-Pick:**
Would like to see less new lines in the method calls with the ");" on the same line as the last parameter.